libraries/Microsoft.Bot.Builder.Expressions/Microsoft.Bot.Builder.Expressions.csproj

```diff
@@ -59,6 +59,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Antlr4.CodeGenerator" Version="4.6.6">
```

Comment:
Probabaly we should create CommonRegex.cs expose a CreateRegex function which return a native Regex object. 
So dialog can use it very naturally.